core/src/test/java/org/testcontainers/containers/GenericContainerTest.java

```diff
@@ -54,6 +53,19 @@ public void shouldReportErrorAfterWait() {
         }
     }
 
+    @Test
+    public void shouldLogImageNameWhenGetDockerImageNameFails() {
+        // A docker image that doesn't exist is enough.  The NotFoundException
+        // that the ContainerFetchException wraps may contain the image name.
+        // This test verifies that the ContainerFetchException itself does.
+        String imageName = "doesnotexist";
```

Comment:
please use `Base58.randomString(8)` or similar, otherwise this test may start failing if you touch `doesnotexist` image on the host machine

Reply:
done in https://github.com/testcontainers/testcontainers-java/pull/2450/commits/47329ba7747df814e8d10615eeb474af8b84cc03